distribution/changelog.txt

```diff
@@ -25,6 +25,7 @@
 - Fix: [#8947] Detection of AVX2 support.
 - Fix: [#8988] Character sprite lookup noticeably slows down drawing.
 - Fix: [#9000] Show correct error message if not enough money available.
+- Fix: [#9152] Spectators being able to modify ride colours.
```

Comment:
Did this really happen in past releases as well? I thought it was a recent regression.

Reply:
Yeah true, ill remove it.

Reply:
As you pointed out on Gitter, it appears to be a regression of #8734 and so warrants a changelog item. :)